src/storage.cc

```diff
@@ -75,9 +75,16 @@ void Storage::EnterNotify() {
   buy_policy.Start();
 
   if (out_commods.size() == 1) {
-    sell_policy.Init(this, &stocks, std::string("stocks"))
+    if (sell_quantity != 0) {
```

Comment:
Don't we get default behavior if `sell_quantity` is 0?  That is, do we need this conditional, or can we always initialize with a sell quantity that may be 0 some times (Default)?

Reply:
Currently, we don't get default behavior with `sell_quantity` = 0. Right now you get an error if you try to run
`sell_policy.Init(this, &stocks, std::string("stocks"), 1e+299, false, sell_quantity)` with 0 in the sell quantity

But you also get other errors due to a an error in `MatlSellPolicy` (fails if facility gets less than `sell_quantity`). With Cyclus [PR #1552](https://github.com/cyclus/cyclus/pull/1552), the if statement is no longer needed. Since #1552 is required for this PR anyway, I'm going to pull the additional if statement out